### PR TITLE
Fix for test_action_crud

### PR DESCRIPTION
**Pull request by quarckster**
In CFME 5.6 you have to wait for loading tag tree a couple seconds.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5ebbc13bfad7be24532ec6bef8efb08651279b8a:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/control/explorer.py [hunk 1]
```diff
@@ -1429,6 +1430,8 @@ def _fill(self, action):
         of forms."""
         fill(self.form, dict(description=self.description, action_type=self.action_type))
         if self.sub_forms[self.action_type] is not None:
+            # Waiting for tree loading
+            sleep(2)
```
Comment (mfalesni) [inline cfme/control/explorer.py]: Can you provide an explanation? We try to avoid sleep everywhere where possible. Possibly you can wait_for_element?

Comment (quarckster) [inline cfme/control/explorer.py]: @mfalesni https://github.com/RedHatQE/cfme_tests/pull/2975/commits/b5107fb7e0fce59ac77c6567626e458befc23cdf

**Comment by dajoRH** [comment]
Lint report for commit b5107fb7e0fce59ac77c6567626e458befc23cdf:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@quarckster 

```
Despite all the workarounds, scrolling to `//button[@data-id='exp_event']/../div/ul/li/a[contains(., "Hourly Timer")]` was unsuccessful.```

**Comment by quarckster** [comment]
@mfalesni 
It seems we have to wait for finishing POST request after selecting option in a form, but now cfme_test doesn't wait for it and tries to pick option from another field before POST request is finished. I believe the problem is here https://github.com/RedHatQE/cfme_tests/blob/master/cfme/web_ui/__init__.py#L1363

**Comment by mfalesni** [comment]
@quarckster I believe the fillings of inputs should be guarded but I know that pete has pushed some change that added a missing wait_for_ajax in the patternfly select. I cna check it out, maybe it is it.

**Comment by mfalesni** [comment]
@quarckster maybe you can try rebasing

**Comment by quarckster** [comment]
Stalled due https://bugzilla.redhat.com/show_bug.cgi?id=1337475

**Comment by mfalesni** [comment]
@quarckster #2992 you said you tried something like this?

**Comment by quarckster** [comment]
After merging #3004 this PR can be closed.

**Comment by quarckster** [comment]
After merging PR #3017 this PR should be closed. Debounce check fixes this test.

**Pull request closed without merging**